### ISSUE TITLE
eSpeak-NG SAPI cannot be used for right-to-left languages in NVDA and JAWS

### ISSUE THREAD

**Issue by amirsol81** [gozaltech/espeak-ng-sapi#6]
@beqabeqa473 First and foremost, thanks for bringing the full power of eSpeak-NG to Windows via this SAPI-oriented attempt.
I've tried adding right-to-left languages, to Persian, via the Configurator. The languages are there and indeed can be selected. But when, say, Persian gets added as a profile, NVDA and JAWS can't use it. These language voices only spell words instead of pronouncing them when selected in JAWS and NVDA. So unless I'm missing something, they can't be used with languages like Arabic and Persian. Any work-arounds or solutions?

**Comment by beqabeqa473** [comment]
Sorry, i am not familiar with these languages, have no idea.

I think they work fine with nvda.

**Comment by amirsol81** [comment]
Unfortunately they don't work with NVDA as I'm an NVDA user.
